activiti-cloud-services-audit-jpa/src/main/java/org/activiti/cloud/services/audit/jpa/controllers/AuditEventsDeleteController.java

```diff
@@ -0,0 +1,65 @@
+package org.activiti.cloud.services.audit.jpa.controllers;
+
+import org.activiti.cloud.api.model.shared.events.CloudRuntimeEvent;
+import org.activiti.cloud.services.audit.api.assembler.EventResourceAssembler;
+import org.activiti.cloud.services.audit.api.converters.APIEventToEntityConverters;
+import org.activiti.cloud.services.audit.api.resources.EventsRelProvider;
+import org.activiti.cloud.services.audit.jpa.events.AuditEventEntity;
+import org.activiti.cloud.services.audit.jpa.repository.EventsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.hateoas.MediaTypes;
+import org.springframework.hateoas.Resource;
+import org.springframework.hateoas.Resources;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+@ConditionalOnProperty(name = "activiti.rest.enable-deletion", havingValue = "true")
+@RestController
+@RequestMapping(
+        value = "/admin/v1/" + EventsRelProvider.COLLECTION_RESOURCE_REL + "/delete",
```

Comment:
this  value = "/admin/v1/" + EventsRelProvider.COLLECTION_RESOURCE_REL + "/delete",

should be
 value = "/admin/v1/" + EventsRelProvider.COLLECTION_RESOURCE_REL + "/",

Reply:
Why the final "/"?

Reply:
no need for it